packages/design-system/src/Documentation/components/CodeBlock.tsx

```diff
@@ -16,7 +20,11 @@ function CodeBlock({ code }: CodeBlockProps) {
       }}
     >
       <Tooltip content="Click here to copy" mouseEnterDelay={0.7}>
-        <Code>{code}</Code>
+        <div>
+          <Code>{code}
+          <IconProvider iconName="copy-control" size="25px" color="" />
```

Comment:
Please use icon button component as I mentioned in the task and not icon component directly. Also it would be great if you fix the alignment issue as well. Text and the button should be aligned. 
![Screenshot 2023-11-06 at 10 18 06 AM](https://github.com/appsmithorg/design-system/assets/87797149/6ac2d47f-0fd4-4f16-8d3c-f5b74bedc610)

Reply:
Also make sure you are using sm size of button.
https://design-system.appsmith.com/?path=/story/design-system-button--icon-button-story&args=startIcon:copy-control;size:sm;kind:tertiary

Reply:
sir can you explain sm size cause when I am using it, it's just won't change like it's shows a very big icon covering half of the screen and I couldn't figure it out
![Screenshot ())](https://github.com/appsmithorg/design-system/assets/104527572/f360cd85-0964-412b-bd10-67c4b77c1701)

Reply:
> Also make sure you are using sm size of button.
> https://design-system.appsmith.com/?path=/story/design-system-button--icon-button-story&args=startIcon:copy-control;size:sm;kind:tertiary

Hi @ALOK9442, sorry for the late response. Like I mentioned here and in the task description, you should be using icon button component and not icon component.

`<Button
  onClick={() => {}}
  size="sm"
  startIcon="copy-control"
/>`